developers_chamber/qa/checks.py

```diff
@@ -173,7 +173,7 @@ class TestPrintStatementsQACheck(RegexPyQACheck):
     """
     name = 'Checking for print statements'
 
-    pattern = r'.*(print *\([^\)]*\)).*'
+    pattern = r'\s*(print *\([^\)]*\)).*'
```

Comment:
Wouldn't this still match the e.g. fingerprint(foo)? But only a part of it?

Reply:
No, that's the point of the fix. Instead of any symbol (the dot), only whitespace is allowed before the `print` keyword.

Reply:
but * is 0 or more, so there doesn't have to be a whitespace, no?
![Screenshot 2022-03-24 at 11 51 38](https://user-images.githubusercontent.com/1315274/159901065-1018b5eb-3435-4db3-aaec-2998adc8fa02.png)

Reply:
Dammit, you are right, I was playing with it and forgot the to add `^`.